cypress/support/step-definitions/table-steps.js

```diff
@@ -0,0 +1,47 @@
+import {
+  rows, checkboxCell, rowByNumber, caption, table,
+} from '../../locators/table';
+
+const THEME_PREFIX = 'carbon-table carbon-table--';
+
+Then('I see {int} records', (records) => {
+  rows().should('have.length', records);
+});
+
+Then('rows are selectable', () => {
+  checkboxCell().should('exist');
+});
+
+Then('rows are not selectable', () => {
+  checkboxCell().should('not.exist');
+});
+
+When('I click row by number {int}', (rowNumber) => {
+  rowByNumber(rowNumber).click();
+});
+
+When('row number {int} is clickable', (rowNumber) => {
+  cy.wait(500); // required because we wait for class change inside iFrame
+  rowByNumber(rowNumber).should('have.class', 'carbon-table-row--clickable');
+});
+
+When('row number {int} is not clickable', (rowNumber) => {
+  cy.wait(500); // required because we wait for class change inside iFrame
+  rowByNumber(rowNumber).should('not.have.class', 'carbon-table-row--clickable');
+});
+
+When('row number {int} is highlighted', (rowNumber) => {
+  rowByNumber(rowNumber).should('have.class', 'carbon-table-row--highlighted');
```

Comment:
we should make a const for a 'carbon-table-row--highlighted'

Reply:
fixed